### PR TITLE
Removing custom Tomcat 7 AMIs

### PR DESCRIPTION
**Pull request by lykabernardo**

### CODE REVIEW

File: Lab_Modules/Module_7/Tomcat_7.json [hunk 1]
```diff
@@ -63,8 +63,14 @@
             "ToPort": "22",
             "CidrIp": "0.0.0.0/0"
           },
-          {
+		  {
```
Comment (dsingh07) [inline Lab_Modules/Module_7/Tomcat_7.json]: @lykabernardo Can we fix the indentation here?

File: Lab_Modules/Module_7/Tomcat_7.json [hunk 2]
```diff
@@ -63,8 +63,14 @@
...
             "IpProtocol": "tcp",
+            "FromPort": "8080",
```
Comment (dsingh07) [inline Lab_Modules/Module_7/Tomcat_7.json]: Do we need to expose 8080 if we're mapping 8080 from the container to the 80 on the host?

Comment (lykabernardo) [reply Lab_Modules/Module_7/Tomcat_7.json]: @dsingh07 you are right, we don't need 8080 exposed. I have removed it from securitygroups.

File: Lab_Modules/Module_7/Tomcat_7.json (hunk 1 above)
Comment (lykabernardo) [reply Lab_Modules/Module_7/Tomcat_7.json]: Indention has been fixed.

**Comment by dsingh07** [comment]
LGTM, thanks @lykabernardo

**Pull request merged**